classes/class-swsales-reports.php

```diff
@@ -28,6 +29,36 @@ public static function add_reports_page() {
 		);
 	}
 
+	/**
+	 * Handles the Sales Export
+	 */
+	public static function sws_stats_csv() {
+		require_once(dirname(__FILE__) . "/../adminpages/report-csv.php");
```

Comment:
Can we use one of these constants instead of dirname here?

https://github.com/strangerstudios/sitewide-sales/blob/dev/sitewide-sales.php#L19-L22

Reply:
Yep - require_once( SWSALES_DIR . "/adminpages/report-csv.php"); done.